packages/debug/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@matanuska/debug",
+  "version": "1.0.0",
+  "description": "Matanuska debug library",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "format": "prettier --write '**.ts'",
+    "lint": "eslint '**.ts' --fix",
+    "test": "tests-of-the-sierra-madre",
```

Comment:
How viable is it to implement tests for `debug`?

Reply:
Not particularly viable - it's a thin wrapper around otel, and doesn't have a good way of mocking it out.